Algorithm.CSharp/CustomUniverseSelectionRegressionAlgorithm.cs

```diff
@@ -0,0 +1,107 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuantConnect.Data;
+using QuantConnect.Interfaces;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// Test algorithm that verifies that securities added through
+    /// <see cref="QCAlgorithm.AddEquity"/> API and universe selection
+    /// both start sending data at the same time
+    /// </summary>
+    public class CustomUniverseSelectionRegressionAlgorithm : QCAlgorithm, IRegressionAlgorithmDefinition
+    {
+        /// <summary>
+        /// Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.
+        /// </summary>
+        public override void Initialize()
+        {
+            SetStartDate(2013, 10, 07);
+            SetEndDate(2013, 10, 11);
+
+            AddEquity("AAPL", Resolution.Daily);
+
+            UniverseSettings.Resolution = Resolution.Daily;
+            AddUniverse(SecurityType.Equity,
+                "SecondUniverse",
+                Resolution.Daily,
+                Market.USA,
+                UniverseSettings,
+                time => new[] { "SPY" });
+        }
+
+        /// <summary>
+        /// OnData event is the primary entry point for your algorithm. Each new data point will be pumped in here.
+        /// </summary>
+        /// <param name="data">Slice object keyed by symbol containing the stock data</param>
+        public override void OnData(Slice data)
```

Comment:
Similar comment here regarding checking the date.

Reply:
Similar response - Actually this regression test was just added to verify/assert the behavior of a custom universe selection (not using coarse) was the same as the coarse selection - This regression test passes in master.